### PR TITLE
New SLE 15 rule service_nftables_enabled

### PR DESCRIPTION
**Pull request by rumch-se**
#### Description:

- _New SLE 15 rule which covers CIS recommendation 3.5.2.9 Ensure nftables service is enabled (Automated)_

#### Rationale:

- At the moment CIS profile for SLE 15 does not include this rule.

### CODE REVIEW

File: linux_os/guide/system/network/network-nftables/service_nftables_enabled/rule.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+documentation_complete: true
+
+prodtype: sle15
+
+title: 'Verify nftables service is enabled'
```
Comment (Mab879) [inline linux_os/guide/system/network/network-nftables/service_nftables_enabled/rule.yml:5]:
```suggestion
title: 'Verify nftables Service Is Enabled'
```

Comment (Mab879) [reply linux_os/guide/system/network/network-nftables/service_nftables_enabled/rule.yml:5]: The title of rules needs to be in title case.

**Review by Mab879 — CHANGES_REQUESTED** [review]
Thanks for the PR; just one minor change.

**Comment by rumch-se** [comment]
Hello @Mab879 
I have made the proposed small change. (https://github.com/ComplianceAsCode/content/pull/10113/commits/b98d058855e3baa99b2eb0444524617ccd090874) 
Have a nice day
Rumen

**Review by Mab879 — APPROVED** [review]

**Comment by Mab879** [comment]
The Automatus failures are expected due to the prodtype.

**Pull request merged**